hyperspy/datasets/artificial_data.py

```diff
@@ -7,13 +7,21 @@
 import numpy as np
 
 from hyperspy.misc.math_tools import check_random_state
-
+from hyperspy.signals import Signal1D,Signal2D,EELSSpectrum
+from hyperspy import components1d,components2d
+from hyperspy.axes import FunctionalDataAxis, UniformDataAxis
 
 ADD_POWERLAW_DOCSTRING = \
 """add_powerlaw : bool
         If True, adds a powerlaw background to the spectrum. Default is False.
     """
 
+ADD_BASELINE_DOCSTRING = \
+"""add_baseline : bool
```

Comment:
Since this is now used only once and not generic enough to be used in other docstrings, should it be moved to the docstring itself?

Reply:
Sure sorry about that